### ISSUE TITLE
PyPI release does not reflect current version on GitHub

### ISSUE THREAD

**Issue by Bigfootonmars** [tbmiller-astro/imcascade#1]
Hello, 
Is it possible that the make_diagnostig_fig() method of the ImcascadeResults class is missing from the PyPI release?
When using 'pip install imcascade' the part of the in-depth example  that uses 'make_diagnostic_fig() does not work cause it cannot find that function (everything else works fine). 
I fixed it by following the other method to install imcascade that you describe (cloning the repository directly and then installing that) but this could be an issue for others too?

**Comment by tbmiller-astro** [comment]
Yes, this was my bad for not updating the pypi package in way too long. My apologies!

The github installation seems like its working fine? if not I have just updated the pypi to v1.1 which is up to date so you should be able to install from there.
